nodes/src/nodes/context_optimizer/optimizer.py

```
@@ -0,0 +1,456 @@
+        'gpt-3.5-turbo': 16385,
+        'claude-opus': 200000,
+        'claude-sonnet': 200000,
+        'claude-haiku': 200000,
+        'claude-3-opus': 200000,
+        'claude-3-sonnet': 200000,
+        'claude-3-haiku': 200000,
+        'gemini-pro': 1000000,
+        'gemini-flash': 1000000,
+        'gemini-2.0-flash': 1000000,
+        'gemini-1.5-pro': 1000000,
+    }
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)